taglibs/colabIceFacelets/src/main/java/org/climatecollaboratorium/facelets/discussions/DiscussionBean.java

```diff
@@ -460,12 +460,12 @@ public void subscribe(ActionEvent e) throws SystemException, PortalException {
             if (liferayUser != null) {
                 if (isSubscribed()) {
                     // user is subscribed, unsubscribe
-                    ActivitiesClient.deleteSubscription(liferayUser.getUserId(), ActivityEntryType
-                            .DISCUSSION.getPrimaryTypeId(), getDiscussionId(), 0, extraData.toString());
                     ;
+                    ActivitiesClient.deleteSubscription(new Long(liferayUser.getUserId()), ActivityEntryType
+                            .DISCUSSION, getDiscussionId(), extraData.toString());
                 } else {
-                    ActivitiesClient.addSubscription(ActivityEntryType.DISCUSSION.getPrimaryTypeId(),
-                            getDiscussionId(), 0, extraData.toString(), liferayUser.getUserId());
+                    ActivitiesClient.addSubscription(new Long(liferayUser.getUserId()), ActivityEntryType.DISCUSSION,
```

Comment:
Broke with the introduction of https://github.com/CCI-MIT/XCoLab/commit/88dc8c63c5111014a0f7d2773323894b14971dd1. @jobachh please take a look at the changes